ytmusicapi/parsers/_utils.py

```diff
@@ -38,6 +38,13 @@ def parse_menu_playlists(data: JsonDict, result: JsonDict) -> None:
             result[watch_key] = watch_id
 
 
+def parse_artists_runs(runs: JsonList) -> JsonList:
```

Comment:
Please don't put new functions in this file. Not exactly one of my brightest ideas to create a `_utils.py` in the first place, but you live and learn I guess.

Feel free to create a new file for it and name it as you see fit (maybe artists.py? runs.py?)